src/main/scala/nl.knaw.dans.easy.bag2deposit/package.scala

```diff
@@ -55,13 +56,208 @@ package object bag2deposit extends DebugEnhancedLogging {
+      //    "<e2><80><94>" -> "—",
+      //    "<e2><80><95>" -> "―",
+      //    "<e2><80><96>" -> "‖",
+      //    "<e2><80><97>" -> "‗",
+      "<e2><80><98>" -> "‘",
+      "<e2><80><99>" -> "’",
+      //    "<e2><80><9a>" -> "‚",
+      //    "<e2><80><9b>" -> "‛",
+      //    "<e2><80><9c>" -> "“",
+      //    "<e2><80><9d>" -> "”",
+      //    "<e2><80><9e>" -> "„",
+      //    "<e2><80><9f>" -> "‟",
+      //    "<e2><80><a0>" -> "†",
+      //    "<e2><80><a1>" -> "‡",
+      //    "<e2><80><a2>" -> "•",
+      //    "<e2><80><a3>" -> "‣",
+      //    "<e2><80><a4>" -> "․",
+      //    "<e2><80><a5>" -> "‥",
+      //    "<e2><80><a6>" -> "…",
+      //    "<e2><80><a7>" -> "‧",
+      //    "<e2><80><b0>" -> "‰",
+      //    "<e2><80><b1>" -> "‱",
+      //    "<e2><80><b2>" -> "′",
+      //    "<e2><80><b3>" -> "″",
+      //    "<e2><80><b4>" -> "‴",
+      //    "<e2><80><b5>" -> "‵",
+      //    "<e2><80><b6>" -> "‶",
+      //    "<e2><80><b7>" -> "‷",
+      //    "<e2><80><b8>" -> "‸",
+      //    "<e2><80><b9>" -> "‹",
+      //    "<e2><80><ba>" -> "›",
+      //    "<e2><80><bb>" -> "※",
+      //    "<e2><80><bc>" -> "‼",
+      //    "<e2><80><bd>" -> "‽",
+      //    "<e2><80><be>" -> "‾",
+      //    "<e2><80><bf>" -> "‿",
+      //    "<e2><81><80>" -> "⁀",
+      //    "<e2><81><81>" -> "⁁",
+      //    "<e2><81><82>" -> "⁂",
+      //    "<e2><81><83>" -> "⁃",
+      //    "<e2><81><84>" -> "⁄",
+      //    "<e2><81><85>" -> "⁅",
+      //    "<e2><81><86>" -> "⁆",
+      //    "<e2><81><87>" -> "⁇",
+      //    "<e2><81><88>" -> "⁈",
+      //    "<e2><81><89>" -> "⁉",
+      //    "<e2><81><8a>" -> "⁊",
+      //    "<e2><81><8b>" -> "⁋",
+      //    "<e2><81><8c>" -> "⁌",
+      //    "<e2><81><8d>" -> "⁍",
+      //    "<e2><81><8e>" -> "⁎",
+      //    "<e2><81><8f>" -> "⁏",
+      //    "<e2><81><90>" -> "⁐",
+      //    "<e2><81><91>" -> "⁑",
+      //    "<e2><81><92>" -> "⁒",
+      //    "<e2><81><93>" -> "⁓",
+      //    "<e2><81><94>" -> "⁔",
+      //    "<e2><81><95>" -> "⁕",
+      //    "<e2><81><96>" -> "⁖",
+      //    "<e2><81><97>" -> "⁗",
+      //    "<e2><81><98>" -> "⁘",
+      //    "<e2><81><99>" -> "⁙",
+      //    "<e2><81><9a>" -> "⁚",
+      //    "<e2><81><9b>" -> "⁛",
+      //    "<e2><81><9c>" -> "⁜",
+      //    "<e2><81><9d>" -> "⁝",
+      //    "<e2><81><9e>" -> "⁞",
+      //    "<e2><81><b0>" -> "⁰",
+      //    "<e2><81><b1>" -> "ⁱ",
+      //    "<e2><81><b4>" -> "⁴",
+      //    "<e2><81><b5>" -> "⁵",
+      //    "<e2><81><b6>" -> "⁶",
+      //    "<e2><81><b7>" -> "⁷",
+      //    "<e2><81><b8>" -> "⁸",
+      //    "<e2><81><b9>" -> "⁹",
+      //    "<e2><81><ba>" -> "⁺",
+      //    "<e2><81><bb>" -> "⁻",
+      //    "<e2><81><bc>" -> "⁼",
+      //    "<e2><81><bd>" -> "⁽",
+      //    "<e2><81><be>" -> "⁾",
+      //    "<e2><81><bf>" -> "ⁿ",
+    )
+    lower.toSeq ++ lower.map(kv => kv._1.toUpperCase -> kv._2).toSeq
```

Comment:
We need a balance between completeness (which encoded characters will we actually meet) and performance (avoid scanning the DDM too many times)

Reply:
I was rather thinking that we should be able to avoid getting this `<..>` code altogether. In EASY the same characters seem to be nicely UTF-8 encoded. So where did it go wrong?

Reply:
@janvanmansum See jira issue DD-812
@lindareijnhoudt maybe you want this table configurable in a CSV?